routes/articles.js

```diff
@@ -0,0 +1,112 @@
+const express = require('express');
+const router = express.Router();
+const articles = require('../db/articles.js');
+
+router
+  .route('/')
+  .get((req, res) => {
+
+    let context = {article: articles.all()};
+
+    res.status(200);
+    res.render('layouts/articles/index', context);
+    return;
+  })
+  .post((req, res) => {
+    if (!checkInputKeys(req.body)) {
+      res.send(`{ "success": false }`);
+      // res.redirect(400, '/articles/new')
+      return;
+    }
+
+    if (articles.checkArticleExists(req.body.title)) {
+      res.send(`{ "success": false }`);
+      console.log('this title already exists!');
```

Comment:
I know this is WIP but just to call it out, `console.log` should generally not be in PR(s) as a best practice.